MapboxNavigation/NavigationViewController.swift

```diff
@@ -18,6 +18,13 @@ public protocol NavigationViewControllerDelegate: VisualInstructionDelegate {
      - parameter canceled: True if the user dismissed the navigation view controller by tapping the Cancel button; false if the navigation view controller dismissed by some other means.
      */
     @objc optional func navigationViewControllerDidDismiss(_ navigationViewController: NavigationViewController, byCanceling canceled: Bool)
```

Comment:
Is this method called? I think it's been replaced by `navigationViewControllerDidFinish`, in which case we should delete it.

Reply:
I'd advocate for including this change in the breaking change log as well, it momentarily tripped me up when the critical logic in my `navigationViewControllerDidDismiss` implementation wasn't being run.

Reply:
This one is different. There are now two slightly confusing delegates.

One is called when the user taps the end button in the navigation UI - `func navigationViewControllerDidDismiss(_ navigationViewController: NavigationViewController, byCanceling canceled: Bool)`

The second one is called when the user arrived at the destination and the navigation ended therefore.

Will mention this in the changelog

Reply:
Sorry to be obtuse, but can you point me to the line of code where `navigationViewControllerDidDismiss` is called?

Reply:
You were right from the beginning, this should be removed, only the car play related methods need to stay